query-connector/src/app/layout.tsx

```diff
@@ -1,6 +1,6 @@
 import "../styles/styles.scss";
 import Header from "./query/components/header/header";
-import Footer from "./footer";
+import Footer from "./query/components/footer/footer";
```

Comment:
Not blocking for this PR, but it seems like an organizational anti-pattern here to have `components` nested within `query` if these components are going to be used in other parts of the app. I don't want to increase our current scope here, but would like to flag this for the team so we can discuss further.

Reply:
Yep, totally makes sense. Will bring it up in a shared forum!